filebeat/docs/how-filebeat-works.asciidoc

```diff
@@ -1,16 +1,25 @@
 [[how-filebeat-works]]
-== How Filebeat works
+== How {beatname_uc} works
 
-In this topic, you learn about the key building blocks of Filebeat and how they work together. Understanding these concepts will help you make informed decisions about configuring Filebeat for specific use cases.
+// REVIWERS: I think we need to modify this topic to be more generic, or we
+// need to explain how different inputs handle the event data differently. It
+// doesn't seem accurate to use the word "file" everywhere as we do here. I
+// tried replacing file with log, but that doesn't work in all situations, so
+// I backed out my changes, and I'm looking for alternatives. Maybe we should
+// make this description more generic, and move the details about reading files
+// to the new topic about the log input. WDYT? I might do that in a separate
+// PR so we can get the refactoring merged.
```

Comment:
+1 on more generic description. As of now filebeat is about collecting logs only, but given as logs can be anything + we might widen the scope of filebeat in the future, `events` or `documents` might be more appropriate.

Reply:
TBH, I think this whole topic needs a rewrite. I'd like to defer this to another PR. I've created an issue to track the work here: https://github.com/elastic/beats/issues/6572